### ISSUE TITLE
Missing files referenced in dist/js/framework/localization/i18n.js

### ISSUE THREAD

**Issue by Andersos** [bitmovin/bitmovin-player-ui#279]
When I updated to the latest version of `bitmovin.playerui.version: 3.9.0` it seems like the new language files are referenced but dont exsist in `bitmovin-player-ui/dist/js/framework/localization/languages/en.json`.

The errors Im getting are:

```
ERROR in ./node_modules/bitmovin-player-ui/dist/js/framework/localization/i18n.js
Module not found: Error: Can't resolve './languages/de.json' in '/Users/anders/dev/repo/node_modules/bitmovin-player-ui/dist/js/framework/localization'
 @ ./node_modules/bitmovin-player-ui/dist/js/framework/localization/i18n.js 17:32-62
 @ ./node_modules/bitmovin-player-ui/dist/js/framework/uimanager.js
 @ ./node_modules/bitmovin-player-ui/dist/js/framework/main.js

ERROR in ./node_modules/bitmovin-player-ui/dist/js/framework/localization/i18n.js
Module not found: Error: Can't resolve './languages/en.json' in '/Users/anders/dev/repo/node_modules/bitmovin-player-ui/dist/js/framework/localization'
 @ ./node_modules/bitmovin-player-ui/dist/js/framework/localization/i18n.js 18:32-62
 @ ./node_modules/bitmovin-player-ui/dist/js/framework/uimanager.js
 @ ./node_modules/bitmovin-player-ui/dist/js/framework/main.js
```

**Comment by Andersos** [comment]
What is the status on this issue? Is the localization ready to use? We have made a Norwegian translation. Should we send a pull request to add that language?

**Comment by stonko1994** [comment]
> What is the status on this issue?

We tracked the issue internally. Once it's finished we will post an update.

> Is the localization ready to use?

Unfortunately, this issue blocks the localization feature when including the UI via `npm`. We will prioritize accordingly.

> We have made a Norwegian translation. Should we send a pull request to add that language?

Yes feel free to contribute 🙂

**Comment by stonko1994** [comment]
This is now fixed with `v3.9.1`
